src/modules/tools/zprobe/ZGridStrategy.cpp

```diff
@@ -327,24 +332,19 @@ bool ZGridStrategy::saveGrid() //std::string *args)
 
 }
 
-bool ZGridStrategy::loadGrid()
+bool ZGridStrategy::loadGrid(std::string args)
```

Comment:
@wolfmanjm I was already wondering about references in the current codebase which is not consistent in using those where it makes sense.